### DISCUSSION TITLE
Full width image

### DISCUSSION THREAD

**Discussion by simonspa** [jpanther/congo#498] [Q&A] [answered]
Dear all,

I'm struggling to add an image to a page that spans the full width of the `simple` page layout, i.e. 100%. Whatever I do, the image width remains at the prose width of the regular page layout with submenu, leaving large margins left and right.

I have read the Tailwind CSS documentation and have experimented with the respective margin classes, i.e. setting

```
{{< figure src="mypicture.jpg" class="mx-0 max-w-full" >}}
```

but to no avail:

![Screenshot from 2023-03-08 07-02-01](https://user-images.githubusercontent.com/1677436/223632095-9b7e8aeb-4865-4c69-925c-1938fc1c3b94.png)

The image is stored as page resource.

Could anyone point me to what I am missing?

Thanks!
Simon

**Comment by shin-sforzando** [comment, marked as answer]
I had the same problem, I created an original shortcode.
https://github.com/sforzando/sfz.dev/blob/main/layouts/shortcodes/figureWidthFull.html

You may not necessarily need to create a shortcode.
Since `w-full` is 100% for the parent, it would be better to use `w-screen`, which is 100vw.

https://tailwindcss.com/docs/width

**Reply by simonspa** [reply]
Awesome, thanks a lot for your help! :)
